relay-server/src/services/store.rs

```diff
@@ -1278,6 +1318,25 @@ struct SpanKafkaMessage<'a> {
     trace_id: &'a str,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+struct MetricsSummaryKafkaMessage<'a> {
+    count: u64,
+    duration_ms: u32,
+    end_timestamp: f64,
+    group: &'a str,
+    is_segment: bool,
+    max: f64,
+    mri: &'a str,
+    min: f64,
+    project_id: u64,
+    retention_days: u16,
+    segment_id: &'a str,
+    span_id: &'a str,
+    sum: f64,
+    tags: BTreeMap<String, String>,
```

Comment:
If we make `SpanMetricsSummary::tags` a non-optional `BTreeMap` with `skip_serializing_if=BTreeMap::is_empty`, we can borrow here:

```suggestion
    sentry_tags: &'a BTreeMap<String, String>>,
```

Reply:
I ended up making it a `RawValue` because I had trouble doing what you were suggesting as:
```
the trait `Deserialize<'_>` is not implemented for `&'a BTreeMap<&'a str, std::string::String>`
```